src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/ConstructorBuilder.Mono.cs

```diff
@@ -325,7 +325,7 @@ public override Module Module
 
         public override string ToString()
         {
-            return "ConstructorBuilder ['" + type.Name + "']";
+            return "Name: " + Name;
```

Comment:
coreclr's ends up doing a bit more than this: it delegates to m_methodBuilder.ToString(), which prints out Name, Attributes, and MethodSignature.  Should this do the same?

Reply:
Yup I'll do that

Reply:
@stephentoub Since we don't use a methodbuilder here in Mono, this complicated to do right after all. I can remove this change from the PR that's just supposed to enable tests, or I can leave it in as at least an improvement. What do you think?

Reply:
Either's fine.  Might be worth tracking the known difference in an issue, but up to you. Thanks.